app/redux/selectors/SetsSelectors.js

```diff
@@ -367,3 +367,37 @@ export const getSlowestDurationEver = (state, set) => {
 };
 
 export const getRevision = (state) => stateRoot(state).revision;
+
+// 1rm
+
+export const getExerciseVelocities = (state, exercise) => {   
```

Comment:
- [ ] rename this as you aren't getting the velocities, you're getting a combination of weight and velocities. idk what you'd call it as im not sure what it is